build.gradle

```diff
@@ -61,6 +61,7 @@ project.ext.externalDependency = [
     'guice': 'com.google.inject:guice:4.2.2',
     'guava': 'com.google.guava:guava:27.0.1-jre',
     'h2': 'com.h2database:h2:1.4.196',
+    'hadoopClient': 'org.apache.hadoop:hadoop-client:3.1.0',
```

Comment:
Just to confirm: Have we run any sec scans since adding these deps?

Reply:
good point

Reply:
looks good